fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
ATcA2wFbgM+ro5MX8wmuLmvEbcH0k9pdqd5ABjU2UBUmCIkSLFoi4uCksAzbHYHb3/0BWaTiR6d2jFyUD8APvp2LunHThIeFspDd4ylqLiEbxetAGDr7n3M+vcbAeVZTHDHECtuDxw+5mb7IRcAHZobGdTVzL4MN+t2OwHo28FE43oG7KWwcFNpQHmljhKWffUqBpOZZgl96NRnHACblkwjL+sYtqJcht35IkajiWXfTCYkLJp6jdvR/fp7Asqz2238779/x2Qy07FLdwYNHVEx72j6ESa/+DT/+uhbCgryeWfKC7Rt14mu3XvTd+CQgPLMJrh1oBm3R5N6wsPOI973SLumBvp3NHIw08PGfW6UgjsHm3G6NUaDYtZqJzqQPCOM7Klwe+DoGUg+6l1Kv/aKqFAItiqW7/Dg9sD1nRUWE+QWwdrkQNLOXc/ybSHtlIc9ad7lXdfZQHQYhFgVi7a6r2pbKM8Z1688R7Mr1TueCY0VfTsYOXTMw+YDHsxGuGWAEYcTCks0q3Z7As67ZYAJtwdST3rYVfb6JTQx0L+jgYOZHjbt9+bdep2pIu/Hne6rW9Eqgls2oe0Lj2OKCCPp7qerbbl2u41p//0nJpOJTl0SGTz0xop5GempTPrzs7z/vy8JDg5hzndfk3X2DPHxDRk7fkJAec7SEjb+8BoGo5mGLfvQOnEsALtWf0hR7jHsJbn0u+nPGIxmkpb/H05HMRGxzekx/LcB5dnsDt7+8AvMJiM9OrVn5PVltfObuaQfP0l4aAgPTRhLUbGNbxcuB8pq53/eDCjPYoLbrrfg9sCR426SUrzvgfbNjFzXxcT+DDfr97oIC4aRfSwAdGhm5O2vbTicV57ndJSwYuarGI1mmrTpQ4fe3tq5Zdk0CrKPYSvOZchtL1JqL2LHms8AOHpgAw9NWh7Q+pVz2G3MmPYGJpOZhE496Td4TMW8YxkpvDvpMV5/fx5BwSFXlXO5tdNshNsGmbE7NYUlmpU7Atv2zEYY1ctbOzPOQHKGt4b17+CtnSFWxdIkD5Gh0K2lwmiEtFOwJz2w2mk2wbj+3tqSdtJTWcuaGOjXwcChY1Vqy0BvbSko0azadeXrV75fd7nh8HE3SVX269d1MbM/w826PU7CghWj+1oq5v3ty5KA3pu1XTeVxUrc/b9Bu13YD+yhaOMqAIK79iL8umEAFK5eim3fLuo9+gzaWYoymjj7v3+CvvLXz263Mf39dzCZTHTs0oPrhoysmHc0/Qh/fem3vPfhLIKCQ/B4PLw9+TkSe/Zn5M13BLR+4tpQXe3VUMCptX7lb3UAACAASURBVJ5aPkFrvUtrvVYp1RoIA17C2+zUqI7NDOzL8PDDRjftmlZ+GGvtXg/zNrnZneqmZYOrW+1Vm5MY2q8nLzz2K9Zu3Vkx3Wg0YjaZMBmNhIcG07JpI/408T7G3ziYYQN6B5zXtbWJXYddfLPSQedWxorp+zPcrEyqrHZhwYrubU2UuqCgOPAD5EM7l9Kux0hG/3IKh3evrJjeb+RERv3iNTr1HcfRQ5s4dng7bboOZcQ9r5B+YCNudwCVF9iyYTX9Bg7h10/9ie1b1lVMd7lcrFwyj8Se/SqmBQUH43DYia0XH/D6dW5hYG+am9lrXXRoXjmeBzM9rNldWdCtZnA4NbPXunC6NEHWwPLaNVEcyNQs2qZp06hy+qYDmsXbNckZHprXV5Q4YPF2zQ+bNTHhga5dpQ7NFMll20L7ppXv+XV7Pczb5N2Bt2hw9R9Y7NTMwN50D3M2uGlfZZs7dFyzbm/leNaLUmTla+ZtchMWrIgM8HirUwsDe9M8fL/ORYdmlet16Ni5r1953g8bXIQHKyJDA8u7GFvaMXZPfLH6Flhm04a19B84mN889RxbN2+omO5yuVi+ZAE9evUFIPVICgf27cFisRIdExNwXkbyMlp0Gsl14ydz9EDltt5t8KMMvPVVWncby8nUzQSHxTLw1lcZctc75GenB5y3avN2hvbvyQuPP8DabVVrpwGzyYjJZCQ8NMRbO399P+NHDLmq2tmllYndR9zMXFVKp5aV5/QOHHXz447KelVkg+9Wl7JkSymHj7sDOogESNm1lLbdRnLjPVNI3Vs5nn1unMjwu1+jQ69xZKZsIrZhG4bf9RpdB95DQo8xfpZ4eZI2raBn/2Hc/5u/sGvr6orpLpeTdcvn0LnHwKvOgMuvnfWjFWfzPcxdf3XbXrum3tq5cKsmoXHl9I37vfV0b7qH5vGKY1mwYKvmh02ahMaB17ROzQ0kp3uYs95F+/Nqy9o9F6ktG12EhwS2fuX79W9/dNC55Xn79XPem5qZqxws2nx1783arpuhvQZQvG09WR//i5DEPhXTgzt0Ifurj8ieMY2Qbr0wBAej7TayPv4XutSBISSwwK0bVtF34FAm/vYFtm8+9zjix6U/0K1n/4ppS+fPonuvAYGtmLimVFeT0xnY7mPe3cDXwFqgnVLqio5OlVITlVLblFLbklZ9dMnHR4RAfrH33+efDLCYvBv6/qOBnUEudyYnl/hY74GFwVA5hL8aP4ZJTz1Cn64d+WHF2orpP6xYy9gbrgs4LypMkVfkXRmPn6ceG6EocWjmbyglOlwRGxFYsS/MPU14dEMAlDKeM6/UXsyB7Yto2204rToP5vTRfayc9SaOknxsRXkB5WVnna1oWqqO57zZXzFq3ARU2TeH1KvfgNfeep9Hn3iO2V9/ElAWQGRo5Xj6O2HkKAWTER4YacZoAJsjsLzwYCiwcdE8swk6NFUcOu6d0SQOxvc3cDqwoTxHZIiqaHY9F9kWOrcwcODo1V0tAogIhfyyq0Hn51R1MltjMipG9zYSEaIIDw3s/RkZosgvvvTr582DMX2NhIdARMjP/xtosrPOEhdXDwBjlW1h7uxvuGncbZSvwYljR2nStDn3P/Rrtm/dhMMR2JuzuOA0oZENgAu3daejmLS9i2necTgAp9K3s/LLp4lt2D6gLIAz2
```

Comment:
I get the error that df3 is not defined.

Reply:
corrected!